.circleci/special-examples.txt

```diff
@@ -0,0 +1,5 @@
+# Special examples are run by themselves on their own parallel runner by CircleCI
```

Comment:
The goal is to take all of the examples that have long timeouts and make them have short timeouts and run by themselves.

Look at a run like this --- https://app.circleci.com/pipelines/github/reach-sh/reach-lang/8703/workflows/258a6e3e-2452-4f99-a61a-1fa33c3b18c2 --- and then look at the ALGO job --- https://app.circleci.com/pipelines/github/reach-sh/reach-lang/8703/workflows/258a6e3e-2452-4f99-a61a-1fa33c3b18c2/jobs/155010 --- and see that there are some that take 7m and some that take 3.5m. My hypothesis is that the 7m ones are those long timeouts AND the tests are flaky.